web/src/pages/calendar/Calendar.vue

```diff
@@ -166,7 +167,17 @@ const inEvent = (date: Date) => {
         const start = parseInt(
             isoDateString(new Date(e.start)).substring(8, 10)
         )
-        const end = parseInt(isoDateString(new Date(e.end)).substring(8, 10))
+
+        let end = parseInt(isoDateString(new Date(e.end)).substring(8, 10))
+
+        if (
+            parseInt(isoDateString(new Date(e.end)).substring(0, 4)) !=
+                parseInt(isoDateString(date).substring(0, 4)) &&
+            parseInt(isoDateString(new Date(e.end)).substring(5, 7)) !=
+                parseInt(isoDateString(date).substring(5, 7))
+        ) {
```

Comment:
This looks confusing. Perhaps assign to variables?

I would also avoid duplicating computational expressions: i.e `isoDateString(new Date(e.end))`. Assign it to a variable as well, and reuse the result from the function.